### PR TITLE
Initialization of the yelloan/node-alpine image

### PR DESCRIPTION
**Pull request by ccoeurderoy**
### Description

- Creation of a new docker image using Alpine

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,37 +1,37 @@
-FROM node:8.11.4
-MAINTAINER Yelloan "devteam@yelloan.com"
+FROM node:carbon-alpine
+LABEL Algoan "devteam@algoan.com"
```
Comment (pierreneveu) [inline Dockerfile]: i think it's dev-team@algoan.com.

File: Dockerfile [hunk 2]
```diff
@@ -45,8 +45,12 @@ RUN set -ex \
   && which docker \
   && (docker version || true)
 
-RUN apt-get install kubectl
-RUN npm -g i npm
-RUN yarn global add nodemon typescript colorguard node-gyp cypress@3.0.3 node-static mocha istanbul bower grunt-cli bower-shrinkwrap-resolver nc
```
Comment (pierreneveu) [inline Dockerfile:50]: is it okay for front application to remove `cypress`?

**Review by pierreneveu — APPROVED** [review]

File: Dockerfile (hunk 2 above)
Comment (ccoeurderoy) [reply Dockerfile:50]: Yes they won't use the alpine image, because it is not support on Alpine

Comment (fabong) [reply Dockerfile:50]: we'll find another way for the front apps

**Pull request merged**